node/parachain/src/service.rs

```diff
@@ -190,12 +178,13 @@ pub fn run_node(
 		let pool = transaction_pool.clone();
 		let network = network.clone();
 
-		Box::new(move |deny_unsafe| {
+		Box::new(move |deny_unsafe, _| {
 			let deps = moonbeam_rpc::FullDeps {
 				client: client.clone(),
 				pool: pool.clone(),
 				deny_unsafe,
 				is_authority,
+				enable_dev_signer: false, // Disable dev signer for now. Maybe later, wire it to the CLI.
```

Comment:
As far as I know, this is the last thing to be addressed before this PR can be merged. @crystalin Shall we support dev signers or not?

Reply:
We decided offline to remove this.